tests/schema/tests.py

```diff
@@ -640,6 +641,22 @@ def test_add_field_binary(self):
         # these two types.
         self.assertIn(columns['bits'][0], ("BinaryField", "TextField"))
 
+    def test_add_field_durationfield(self):
+        """
+        Test adding a duration field with a default error doesn't error (#33358)
+        """
+        # Create the table
+        with connection.schema_editor() as editor:
+            editor.create_model(Author)
+            # Add the new field
+            new_field = DurationField(default=datetime.timedelta(minutes=5))
+            new_field.set_attributes_from_name('duration')
```

Comment:
`editor` is not require to create an instance of `DurationField` or to call `column_classes()` that's why I kept these out of the context manager.

Reply:
fair point. hasty copy-pasta.